### ISSUE TITLE
Edits to section labels in Navigator do not trigger UPDATE save

### ISSUE THREAD

**Issue by lisaschiffli** [elementor/elementor#10772]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X ] The issue still exists against the latest stable version of Elementor.

## Description
If you only make edits to the sections/columns/etc labels in the Navigator and don't make any changes to any elements, the UPDATE button is not triggered to Save changes. If you exit the editor, no changes to Navigator are saved. Work around is to make a small edit to an element and then reverse it to trigger UPDATE save.

## Steps to reproduce
Open Elementor Editor on a page. Make a change to a section label in the Navigator. The UPDATE button is not triggered for a save. If you exit the page, the changes to Navigator are not saved.

## Isolating the problem

- [X ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active. (Astra theme)
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by jazir5** [comment]
I'm running into this everywhere as well, i have to change padding/margins, then revert the change to trigger the update button. Can confirm this is an active issue and i'm running Hello Theme, and Elementor 2.9.6 and Elementor Pro 2.8.5

**Comment by DorShahar** [comment]
Hey all,

This issue has been resolved in Elementor v3.2.0

Feel free to update!
